trust-center-agent/src/main/java/care/smith/fts/tca/consent/GicsPropertyNotEmptyCondition.java

```diff
@@ -0,0 +1,29 @@
+package care.smith.fts.tca.consent;
+
+import java.util.Arrays;
+import java.util.stream.StreamSupport;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.EnumerablePropertySource;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+@Slf4j
+public class GicsPropertyNotEmptyCondition implements Condition {
```

Comment:
Having a negation in the class name seems cumbersome, can we name it `GicsConfiguredCondition` or even `GicsConfigured` this would read nicely in Conditional(GicsConfigured...)?